entity-service/internal/server/routes.go

```
@@ -278,7 +278,11 @@ func NewRouter(db *pgxpool.Pool, cfg *config.Config) http.Handler {
 	mux.HandleFunc("DELETE /attachments/{id}", caseHandler.DeleteCaseAttachment)
 	mux.HandleFunc("POST /cases/{id}/tags", caseHandler.AddCaseTag)
 	mux.HandleFunc("DELETE /cases/{id}/tags/{tagId}", caseHandler.RemoveCaseTag)
-	mux.HandleFunc("GET /tags/search", caseHandler.SearchTags)
+	mux.HandleFunc("POST /tags/search", caseHandler.SearchTags)
+	// Deprecated: the query-parameter form of tag search, kept for one release
+	// so callers can be rolled out independently of this service. Remove it
+	// (and CaseHandler.SearchTagsQuery) once they are all on the POST.
+	mux.HandleFunc("GET /tags/search", caseHandler.SearchTagsQuery)
```

golangci-lint (2.12.2)

[error] SA1019: caseHandler.SearchTagsQuery is deprecated: use POST /tags/search instead. This alias exists only so the services in front of this one can be rolled out over the following release instead of having to deploy in lockstep with it; a caller still on the GET would otherwise get a 405. Delete it once that release has shipped. 

(staticcheck)